src/branch/switch.rs

```diff
@@ -16,7 +17,12 @@ pub async fn main(
         .await?;
 
```

Comment:
Just remembered: let's also check that we are not already on this branch, so you don't get messages like:

```
Switching from `hello` to `hello`
```